sdk/cosmosdb/arm-cosmosdb/README.md

```diff
@@ -2,11 +2,11 @@
 
 This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure CosmosDBManagement client.
 
-Azure Cosmos DB Database Service Resource Provider REST API
+Azure Cosmos DB Chaos Fault REST API
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/cosmosdb/arm-cosmosdb) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/arm-cosmosdb) |
-[API reference documentation](https://learn.microsoft.com/javascript/api/@azure/arm-cosmosdb) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-cosmosdb?view=azure-node-preview) |
```

Comment:
why do we have the change here? I feel like the learn.microsoft.com is the right link?